src/item.cpp

```diff
@@ -1523,6 +1523,11 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
                 }
             }
         }
+        
+        if( is_gun() && has_flag( "FIRE_TWOHAND" ) ) {
+            info.push_back( iteminfo( "DESCRIPTION",
+                                      _( "* This weapon needs <info>two free hands</info> to fire." ) ) );
```

Comment:
Would it be possible to do this in some tricky or slow way using just one hand? Like, putting your repeater on the ground, holding it with your leg and pumping it with your hand, or putting your heavy crossbow on the ground and slowly loading it with one hand?